internal/stdlib/syscall/exec_windows.go

```diff
@@ -263,10 +267,10 @@ func StartProcess(argv0 string, argv []string, attr *ProcAttr) (pid int, handle
 	}
 
 	if len(attr.Files) > 3 {
-		return 0, 0, EWINDOWS
+		return 0, 0, syscall.EINVAL
```

Comment:
same question about `EWINDOWS`

Reply:
I'm not sure how these got swapped.. but good catch